test/argparse.cpp

```diff
@@ -0,0 +1,38 @@
+#include <gtest/gtest.h>
+
+#include "cap32.h"
+#include "argparse.h"
+
+TEST(ArgParseTest, parseArgsNoArg)
```

Comment:
Not mandatory:
It would be cool to also be able to tests flags (especially if we plan to add more) but I recon this would require some over-engineering (what I can think of is a ArgParser class with a virtual Exit method and injectable streams in replacement of cout/cerr).